ide/css.visual/src/org/netbeans/modules/css/visual/SelectorsGroupEditor.java

```diff
@@ -111,7 +111,7 @@ private void initComponents() {
         selectElementComboBox = new javax.swing.JComboBox();
         selectClassRadioButton = new javax.swing.JRadioButton();
         selectClassTextField = new javax.swing.JTextField();
-        selectElelmentIdRadioButton = new javax.swing.JRadioButton();
+        selectElementIdRadioButton = new javax.swing.JRadioButton();
```

Comment:
You're changing an UI variable name, did you change the corresponding .form file as well?

Reply:
8000efbc fixed